clm_data/README.md

```diff
@@ -10,27 +10,53 @@ To recreate the artifact:
 2. Activate the new virtual env
 3. In the same terminal run `pip install -r requirements.txt`
-4. In the same terminal run `julia --project=. create_artifact.jl`
+4. In the same terminal run `julia --project create_artifact.jl`
 
 ### Requirements
 - Python >=3.8
 
 ## Input Files
 
 ### 1. `surfdata_0.9x1.25_16pfts__CMIP6_simyr2000_c170616.nc`
-The netCDF file includes comprehensive environmental data with a focus on vegetation represented through 16 different Plant Functional Types (PFTs). These PFTs play a crucial role in modeling biophysical processes and ecosystem functions within CLM simulations.
+The netCDF file includes comprehensive environmental data with a focus on vegetation represented through 15 different Plant Functional Types (PFTs). These PFTs play a crucial role in modeling biophysical processes and ecosystem functions within CLM simulations.
+
+-**Plant Functional Types**
+  - `not_vegetated`
+  - `needleleaf_evergreen_temperate_tree`
+  - `needleleaf_evergreen_boreal_tree`
+  - `needleleaf_deciduous_boreal_tree`
+  - `broadleaf_evergreen_tropical_tree`
+  - `broadleaf_evergreen_temperate_tree`
+  - `broadleaf_deciduous_tropical_tree`
+  - `broadleaf_deciduous_temperate_tree`
+  - `broadleaf_deciduous_boreal_tree`
+  - `broadleaf_evergreen_shrub`
+  - `broadleaf_deciduous_temperate_shrub`
+  - `broadleaf_deciduous_boreal_shrub`
+  - `c3_arctic_grass`
+  - `c3_non-arctic_grass`
+  - `c4_grass`
 
 - **Key Variables Related to PFTs**
     - `PCT_NATVEG`: This variable gives the percentage of natural vegetation cover across the land units, essential for assessing non-agricultural land cover.
-    - `PCT_NAT_PFT`: Indicates the percentage composition of each PFT (16) within natural vegetation land units, providing a granular look at vegetation distribution.
+    - `PCT_NAT_PFT`: Indicates the percentage composition of each PFT (15) within natural vegetation land units, providing a granular look at vegetation distribution.
 
 These PFT variables are pivotal for simulating how natural ecosystems respond to historical climatic conditions and can be used to project changes in vegetation patterns due to climatic shifts.
 
 - **Data Download** The dataset can be downloaded from the official input data repository for the Community Climate System Model:
 [CCSM Input Data Repository](https://svn-ccsm-inputdata.cgd.ucar.edu/trunk/inputdata/lnd/clm2/surfdata_map/)
 
 ### 2. `pft-physiology.c110225.nc`
-Contains physiological parameters for each of the 21 PFTs in a previous version of CLM, when vcmax25 did not depend on nitrogen content.
+Contains physiological parameters for each of the 21 PFTs in a previous version of CLM, when maximum rate of carboxylation did not depend on nitrogen content.
+
+-**Plant Functional Types**
+In addition to the PFTs in `surfdata_0.9x1.25_16pfts__CMIP6_simyr2000_c170616.nc`, this file also includes:
+  - `c3_crop`
+  - `c3_irrigated`
+  - `corn`
+  - `spring_wheat`
+  - `winter_wheat`
+  - `soybean`
```

Comment:
Should this just link to page in the CLM5 documentation?

Reply:
Yes. Maybe point to the variable names and add a link to the CLM5 doc.

Reply:
I'm slightly confused. What exactly do you mean by point to the variable names?

Should I remove the list of the PFT names?

Reply:
No. Just make sure they are correct and updated.